### PR TITLE
TVB-1837, 2282

### PR DESCRIPTION
**Pull request by danvladpop**

### CODE REVIEW

File: tvb/analyzers/fft.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def evaluate(self):
             time_series = numpy.concatenate(segments, axis=4)
         else:
             self.segment_length = time_series_length
-            time_series = self.time_series.data[:, :, :, numpy.newaxis]
+            time_series = self.time_series.data[:, :, :, :, numpy.newaxis]
```
Comment (liadomide) [inline tvb/analyzers/fft.py:124]: Shouldn't we have a similar change at line 109 above ?

Comment (maedoc) [reply tvb/analyzers/fft.py:124]: This change does not make sense given the other changes in this PR.

Comment (liadomide) [reply tvb/analyzers/fft.py:124]: The pull request covers 2 issues: TVB-1837 and bug TVB-2282.
This line is the only change related to TVB-2282

**Review by liadomide — APPROVED** [review]

File: tvb/simulator/common.py [hunk 2]
```diff
@@ -264,3 +265,33 @@ def zip_directory(path, zip_file):
...
+            if s_unit in ('s', 'sec'):
+                total_milliseconds = s_time * 1000
+            elif s_unit in ('m', 'min'):
+                total_milliseconds = s_time * 60000
+            elif s_unit in ('hr', 'h'):
+                total_milliseconds = s_time * 3600000
+            elif s_unit in ('ms'):
+                total_milliseconds = s_time
+        except:
+            return 0
```
Comment (liadomide) [inline tvb/simulator/common.py]: Why not throw an exception here ?

Comment (maedoc) [reply tvb/simulator/common.py]: agreed, this should be something liek
```python
match = re_times.match(duration)
if not match:
    raise ValueError("unable to parse duration %r" % duration)
...
```

**Pull request merged**